### PR TITLE
Change UI for Student stage

### LINKED ISSUE #95: Change UI Display for Student List
Student stage currently displays only the student list

The list of courses are not displayed at all, which may be slightly confusing. A sidebar (side display of courses) would be helpful for the user to still be able to keep in track of how many courses there are while viewing students in a certain course.

### PR DESCRIPTION
**Pull request by phiphi-tan**
Fixes #95 

checkstyleMain ran and passed.

### CODE REVIEW

File: src/main/java/seedu/address/ui/CombinedPanel.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.StackPane;
+import seedu.address.model.course.Course;
+import seedu.address.model.person.Student;
+
+/**
+ * CombinedPanel containing a side panel of Courses and a main panel of Students
```
Comment (sunzihan23) [inline src/main/java/seedu/address/ui/CombinedPanel.java:11]: Minor issue but missing a full stop here

File: src/main/resources/view/CombinedPanel.fxml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+            <Insets bottom="5.0" left="5.0" right="5.0" top="5.0" />
+         </padding>
+      </StackPane>
+      <StackPane fx:id="studentListPlaceholder" prefHeight="150.0" prefWidth="575.0">
+         <padding>
+            <Insets bottom="5.0" left="5.0" right="5.0" top="5.0" />
+         </padding>
+      </StackPane>
+   </children>
+</HBox>
```
Comment (gongg21) [inline src/main/resources/view/CombinedPanel.fxml:21]: LGTM

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 3]
```diff
@@ -229,11 +228,9 @@ private CommandResult executeCommand(String commandText) throws CommandException
 
             Stages current = StageManager.getStage();
             if (current == Stages.COURSE) {
-                System.out.println("Stage is COURSE");
-                loadStudentListPanel();
+                loadCombinedPanel();
```
Comment (gongg21) [inline src/main/java/seedu/address/ui/MainWindow.java:231]: LGTM

File: src/main/java/seedu/address/ui/CombinedPanel.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+ */
+public class CombinedPanel extends UiPart<Region> {
```
Comment (gongg21) [inline src/main/java/seedu/address/ui/CombinedPanel.java:13]: Nice job implementing the CombinedPanel 👍

**Review by gongg21 — COMMENTED** [review]
Consistent and well implemented to match project idea.

**Pull request merged**